ckanext/switzerland/helpers.py

```diff
@@ -533,3 +533,30 @@ def showcase_types():
 def get_showcase_type_name(showcase_type):
     type_string = showcase_types_mapping.get(showcase_type, showcase_type)
     return get_localized_value(parse_json(type_string))
+
+
+def package_has_group(group_name, groups):
```

Comment:
I am sort of struggeling with the name of that function here: it looks more like a `group_name_in_groups` function. I assume that you use on a package, but since that package does not appear in the function itself, I would suggest to name it differently.